test/users/get_users.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const app = require('../../app');
+
+describe('Tests sign in', () => {
+  const paginationQuery = {
+    page: '0',
+    zise: '3'
+  };
+
+  const user = {
+    name: 'pepito',
+    last_name: 'Perez',
+    mail: 'pepito@wolox.co',
+    pass: 'PassPepit0'
+  };
+
+  const login = {
+    mail: 'pepito@wolox.co',
+    pass: 'PassPepit0'
+  };
+
+  test('Successful transaction', async done => {
+    await request(app)
+      .post('/users')
+      .send(user)
+      .expect(201);
+
+    const logInData = await request(app)
+      .post('/users/sessions')
+      .send(login)
+      .expect(200);
```

Comment:
Do not rely on other features to test this endpoint. You can mock the session and create the user directly in the database with a factory.